### PR TITLE
Tailmods

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

File: solar/solar.py [hunk 1]
```diff
@@ -110,22 +118,20 @@ def setup(self, sp=False):
             self.components.extend([self.fuselage])
             constraints.extend([
                 Volbatt <= self.fuselage["\\mathcal{V}"],
-                Wwing >= self.wing.W + self.solarcells["W"],
-                Wcent >= (Wpay + Wavn + self.emp.W
-                          + self.motor.W + self.fuselage["W"]
-                          + self.battery.W),
+                Wwing >= self.wing.W + self.solarcells.W,
+                Wcent >= (Wpay + Wavn + self.emp.W + self.motor.W
+                          + self.fuselage["W"] + self.battery.W),
```
Comment (bqpd) [inline solar/solar.py]: self.fuselage.W?

File: solar/solar.py [hunk 2]
```diff
@@ -110,22 +118,20 @@ def setup(self, sp=False):
...
-                          sum(summing_vars([self.emp, self.motor], "W"))),
+                Wwing >= sum([c.W for c in [self.wing, self.battery,
+                                            self.solarcells]]),
+                Wcent >= Wpay + Wavn + self.emp.W + self.motor.W,
                 Volbatt <= cmac**2*0.5*tau*b
                 ])
 
         constraints.extend([Wtotal >= (
-            Wpay + Wavn + sum(summing_vars(self.components, "W")))])
+            Wpay + Wavn + sum([c.W for c in self.components]))])
```
Comment (bqpd) [inline solar/solar.py]: yeehaw

File: solar/solar.py (hunk 1 above)
Comment (mjburton11) [reply solar/solar.py]: I just haven't put fuselage into parsing yet... coming soon!

File: solar/solar.py (hunk 2 above)
Comment (mjburton11) [reply solar/solar.py]: I know! about time I got rid of this

**Pull request merged**